src/main/java/com/bnkrll/controller/SessionController.java

```diff
@@ -35,4 +35,18 @@ public List<Session> getPreviousSessions(
                     int last) {
         return sessionRepository.getLastSessions(last);
     }
+
+    @PatchMapping
+    public ResponseEntity<Session> updateSession(@RequestBody Session session){
```

Comment:
I think Void is fine here. Usually state changing behavior should return void since they are commands. If anything were to go wrong then we could have a status code that the UI can response to with.

Reply:
Done !